sdk/include/reactos/shlobj_undoc.h

```diff
@@ -156,8 +164,8 @@ DECLARE_INTERFACE_(IGlobalFolderSettings, IUnknown)
 	STDMETHOD_(ULONG,AddRef)(THIS) PURE;
 	STDMETHOD_(ULONG,Release)(THIS) PURE;
     /*** IGlobalFolderSettings ***/
-	STDMETHOD(Get)(THIS_ struct DEFFOLDERSETTINGS *buffer, int theSize) PURE;
-	STDMETHOD(Set)(THIS_ const struct DEFFOLDERSETTINGS *buffer, int theSize, unsigned int param14) PURE;
+    STDMETHOD(Get)(THIS_ DEFFOLDERSETTINGS *pFDS, UINT cb) PURE;
+    STDMETHOD(Set)(THIS_ const DEFFOLDERSETTINGS *pFDS, UINT cb, UINT unknown) PURE;
```

Comment:
Since you modified the indentation here, could you fix it also for the three IUnknown methods above? Otherwise this makes the whole thing less consistent.